www/js/control/general-settings.js

```diff
@@ -347,57 +280,28 @@ angular.module('emission.main.control',['emission.services',
         $scope.refreshScreen();
     });
 
+    //in ProfileSettings in DevZone
     $scope.refreshScreen = function() {
         console.log("Refreshing screen");
         $scope.settings = {};
-        $scope.settings.collect = {};
         $scope.settings.sync = {};
-        $scope.settings.notification = {};
         $scope.settings.auth = {};
         $scope.settings.connect = {};
         $scope.settings.clientAppVer = ClientStats.getAppVersion();
         $scope.getConnectURL();
-        $scope.getCollectionSettings();
         $scope.getSyncSettings();
         $scope.getOPCode();
-        $scope.getState().then($scope.isTrackingOn).then(function(isTracking) {
-            $scope.$apply(function() {
-                console.log("Setting settings.collect.trackingOn = "+isTracking);
-                $scope.settings.collect.trackingOn = isTracking;
-            });
-        });
-        KVStore.get("OP_GEOFENCE_CFG").then(function(storedCfg) {
-            $scope.$apply(function() {
-                if (storedCfg == null) {
-                    console.log("Setting settings.collect.experimentalGeofenceOn = false");
-                    $scope.settings.collect.experimentalGeofenceOn = false;
-                } else {
-                    console.log("Setting settings.collect.experimentalGeofenceOn = true");
-                    $scope.settings.collect.experimentalGeofenceOn = true;
-                }
-            });
-        });
-        if ($scope.ui_config.reminderSchemes) {
-            NotificationScheduler.getReminderPrefs().then((prefs) => {
-                $scope.$apply(() => {
-                    const m = moment(prefs.reminder_time_of_day, 'HH:mm');
-                    // defining data used to populate the upcoming display
-                    $scope.settings.notification.scheduledNotifs = NotificationScheduler.scheduledNotifs;
-                    $scope.settings.notification.prefReminderTimeVal = m.toDate();
-                    $scope.settings.notification.prefReminderTimeOnLoad = prefs.reminder_time_of_day;
-                    $scope.updatePrefReminderTime(false); // update the displayed time
-                });
-            });
-        }
         $scope.getUserData();
     };
 
-    $scope.copyToClipboard = (textToCopy) => {
-        navigator.clipboard.writeText(textToCopy).then(() => {
-            ionicToast.show('{Copied to clipboard!}', 'bottom', false, 2000);
-        });
-    }
+    //this feature has been eliminated (as of right now)
+    // $scope.copyToClipboard = (textToCopy) => {
+    //     navigator.clipboard.writeText(textToCopy).then(() => {
+    //         ionicToast.show('{Copied to clipboard!}', 'bottom', false, 2000);
+    //     });
+    // }  
```

Comment:
Do we want to restore it? It seems like it would be a good way to get the opcode in text. But it seems like it is a bit of an easter egg since it is non-obvious that clicking it will copy the opcode. And the button in that row is already used for logout.

We could split it into two rows in the future - one with the opcode and a "copy button" and one with the text "logout" and the logout button.

Thoughts?

Reply:
It only worked on iOS and yeah, I used it as an Easter egg / dev convenience feature.
To copy text on Android we would need an extra permission

Reply:
If we wanted to split into two rows, I could absolutely do that -- the copy function itself would be simple, it was just that I couldn't make the description have a separate "onClick" from the row itself. 

If Android requires other permissions is that something we want to pursue? We already have a way to save the opcode as a document, so I don't think it's critical to users.

One way I can think of to keep it as an "easter egg" would be to display the opcode in one row, with no function, and maybe a "profile" icon, and then have the logout row just below that. Then the copy click can remain a dev/ios bonus. @JGreenlee @shankari what do you think of 2 rows with the function only available to iOS?